src/PathResolver/OperationPathResolverInterface.php

```diff
@@ -27,9 +27,6 @@
      * @param array       $operation         The operation metadata
      * @param string|bool $operationType     One of the constants defined in ApiPlatform\Core\Api\OperationType
      *                                       If the property is a boolean, true represents OperationType::COLLECTION, false is for OperationType::ITEM
-     * @param string      $operationName     The operation name
```

Comment:
false positive, must be fixed

Reply:
If you fix it then the problem will be recurrent if we keep the `no_superfluous_phpdoc_tags` rule in our config.